mobile/android/android-device.ts

```diff
@@ -52,14 +53,16 @@ export class AndroidDevice implements Mobile.IDevice {
 	private version: string;
 	private vendor: string;
 	private _tmpRoots: IStringDictionary = {};
-	private _installedApplications: string[];
+    private _installedApplications: string[];
+    private defaultNodeInspectorUrl: string = "http://127.0.0.1:8080/debug";
```

Comment:
@teobugslayer As discussed earlier the port is hard coded for this version
@Fatme done with https://github.com/telerik/mobile-cli-lib/commit/1a8d23f43fcd26503d703143ff750445c94f88a8#diff-63f4603786adf475e860b557e6e5ea98R57